### PR TITLE
myTBA GameDay

### PR DESCRIPTION
**Pull request by fangeugene**
GameDay's favorite teams now pull from and sync with myTBA!

### CODE REVIEW

File: controllers/ajax_controller.py [hunk 1]
```diff
@@ -3,16 +3,55 @@
...
+
+        favorites = Favorite.query(ancestor=ndb.Key(Account, self.user_bundle.user.user_id())).fetch()
+        self.response.out.write(json.dumps([ModelToDict.favoriteConverter(fav) for fav in favorites]))
+
+    def post(self):
+        if not self.user_bundle.user:
+            self.response.set_status(401)
+            return
+
+        action = self.request.get("action")  # 'add' or 'delete'
```
Comment (gregmarra) [inline controllers/ajax_controller.py]: Why did you choose this over separate `AccountFavoritesAddHandler` and `AccountFavoritesDeleteHandler`?

Comment (fangeugene) [inline controllers/ajax_controller.py]: No real reason. Less duplication? =\

Comment (fangeugene) [inline controllers/ajax_controller.py]: I'll split it.

**Comment by synth3tk** [comment]
Woot! :smile:

**Pull request merged**